docs/manual/guide/production/ConductR.md

```diff
@@ -26,63 +48,92 @@ The above creates what is known as a "bundle". A bundle is the unit of deploymen
 
 ## Loading and running your services during development
 
-From a development perspective ConductR provides a "sandbox" environment where you can start it and then test your bundle long before it goes to production. ConductR's sandbox also permits you to [debug Lagom services](https://github.com/typesafehub/sbt-conductr-sandbox#debugging-application-in-conductr-sandbox) from your IDE when running within its cluster.
+In the previous [Installing ConductR sandbox](#Installing-ConductR-sandbox) section we have already installed the `conductr-cli`. The CLI can be used on the terminal to communicate with the ConductR cluster. 
 
-To access ConductR's sandbox from within the activator console first added the following sbt plugin to your `project/plugins.sbt`:
+To use the same commands within the activator console add the following sbt plugin to your `project/plugins.sbt`:
```

Comment:
Why having two ways of doing the same thing? We should be opinionated and provide one path that works. Providing two leads to confusion (what are the differences? when should I prefer one over the other?)